packages/stencil/components/tabs/stencil.config.ts

```diff
@@ -0,0 +1,24 @@
+import { Config } from '@stencil/core';
+import { getStencilConfig } from '@ovhcloud/ods-stencil/libraries/stencil-core';
+import * as jestConfig from './jest.config';
+
+const args = process.argv.slice(2);
+
+export const config: Config = getStencilConfig({
+  namespace: 'osds-tabs',
+  args,
+  jestConfig: jestConfig.default,
+  reactOutput: {
+    componentCorePackage: '@ovhcloud/ods-stencil/components/tabs',
+    excludeComponents: ['osds-link', 'osds-text'],
+  },
+  vueOutput: {
+    componentCorePackage: '@ovhcloud/ods-stencil/components/tabs',
```

Comment:
You should exclude link and text from Vue too:     
excludeComponents: ['osds-link', 'osds-text'],

Reply:
👍